prow/github/client.go

```diff
@@ -362,7 +362,7 @@ func (c *Client) DeleteStaleComments(org, repo string, number int, comments []Is
 // function should accept that populated slice for each page of
 // results.  An error is returned if encountered in making calls to
 // github or marshalling objects.
-func (c *Client) readPaginatedResults(path string, newObj func() interface{}, accumulate func(interface{})) error {
+func (c *Client) readPaginatedResults(path, accept string, newObj func() interface{}, accumulate func(interface{})) error {
```

Comment:
Seems not used.

Reply:
It is used in a couple places below to specify an accept header for API calls that require it.